.metadata/scraper.header.txt

```diff
@@ -0,0 +1,2 @@
+// Remove when https://github.com/microsoft/win32metadata/commit/273d404e3e851e31766a884c1bb9a14d3606570f is in a released WinmdGenerator
+using Windows.Win32.Graphics.Dxgi.Common;
```

Comment:
As per https://github.com/microsoft/DirectX-Headers/pull/82#discussion_r1028605900 this entire file should be removed when upgrading to the latest `WinmdGenerator`, where this `using` has also been added to the global `scraper.header.txt`.

Reply:
I just published a new win32metadata update with these changes.

Reply:
Yup, seen that, I'm subscribed to the releases. Thanks! Now we just need a solution for https://github.com/microsoft/win32metadata/issues/1369 and some more digging into `d3d12shader`/`d3d12compatibility`/`d3d12sdklayers`.